frameworks/src/foundry/mod.rs

```diff
@@ -145,9 +145,9 @@ impl ParseLow for Foundry {
         "assert*",
         "vm.expectEmit",
         "vm.expectRevert",
-        "vm.prank",
-        "vm.startPrank",
-        "vm.stopPrank",
```

Comment:
I included `prank`, etc. because, in many of my experiments, these were used to send from an address with less privilege. So removing calls to `prank`, etc. meant sending from a more privileged address, and thus lots of false positives.

I am open to removing them from this list. But rather than remove them outright, please comment them out and add a note indicating that we may want to revisit them.

Also, could you please update the README? https://github.com/trailofbits/necessist/blob/45c1abba36889159badc88ec0dc5c6c57fa15868/README.md?plain=1#L162